src/services/registration/registration-router.test.ts

```diff
@@ -37,6 +37,10 @@ const APPLICATION = {
     hackOutreach: [HackOutreach.INSTAGRAM],
 } satisfies RegistrationApplicationRequest;
 
+const APPLICATION_INVALID_EMAIL = Object.assign({}, APPLICATION, {
```

Comment:
This is a weird syntax. Use spread operator:
= { ...APPLICATION, emailAddress: "whatever"}